src/frontend/frontend/views/dashboard_views.py

```
@@ -44,18 +44,23 @@ def static_view(cls):
         try:
             trending_clusters = DataPersistenceLayer().get_objects(TrendingCluster)
             dashboard_config = current_user.profile.dashboard
+            from frontend.views.story_views import StoryView
+
+            saved_filters = StoryView.get_saved_filter_links()
         except HTTPException:
             raise
         except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)